### ISSUE TITLE
Can't Preview

### ISSUE THREAD

**Issue by tlberglund** [asciidoctor/atom-asciidoc-preview#65]
Using 0.4.3 of the package and Atom 0.177.0, I can't run the `asciidoctor-preview:toggle` command. I noticed this in the console when I started up:

```
Failed to activate package named 'asciidoc-preview' Error: Cannot find module '/Applications/Atom.app/Contents/Resources/app/node_modules/highlights'
  at Module._resolveFilename (module.js:351:15)
  at Function.Module._resolveFilename (/Applications/Atom.app/Contents/Resources/app/src/module-cache.js:383:52)
  at Function.Module._load (module.js:293:25)
  at Module.require (module.js:380:17)
  at require (module.js:399:17)
  at Object.<anonymous> (/Users/tlberglund/.atom/packages/asciidoc-preview/lib/renderer.coffee:8:14)
  at Object.<anonymous> (/Users/tlberglund/.atom/packages/asciidoc-preview/lib/renderer.coffee:1:1)
  at Module._compile (module.js:475:26)
  at Object.requireCoffeeScript [as .coffee] (/Applications/Atom.app/Contents/Resources/app/src/coffee-cache.js:74:19)
  at Module.load (/Applications/Atom.app/Contents/Resources/app/node_modules/coffee-script/lib/coffee-script/register.js:45:36)
  at Function.Module._load (module.js:325:12)
  at Module.require (module.js:380:17)
  at require (module.js:399:17)
  at Object.<anonymous> (/Users/tlberglund/.atom/packages/asciidoc-preview/lib/asciidoc-preview-view.coffee:7:12)
  at Object.<anonymous> (/Users/tlberglund/.atom/packages/asciidoc-preview/lib/asciidoc-preview-view.coffee:1:1)
  at Module._compile (module.js:475:26)
  at Object.requireCoffeeScript [as .coffee] (/Applications/Atom.app/Contents/Resources/app/src/coffee-cache.js:74:19)
  at Module.load (/Applications/Atom.app/Contents/Resources/app/node_modules/coffee-script/lib/coffee-script/register.js:45:36)
  at Function.Module._load (module.js:325:12)
  at Module.require (module.js:380:17)
  at require (module.js:399:17)
  at Object.<anonymous> (/Users/tlberglund/.atom/packages/asciidoc-preview/lib/asciidoc-preview.coffee:3:23)
  at Object.<anonymous> (/Users/tlberglund/.atom/packages/asciidoc-preview/lib/asciidoc-preview.coffee:1:1)
  at Module._compile (module.js:475:26)
  at Object.requireCoffeeScript [as .coffee] (/Applications/Atom.app/Contents/Resources/app/src/coffee-cache.js:74:19)
  at Module.load (/Applications/Atom.app/Contents/Resources/app/node_modules/coffee-script/lib/coffee-script/register.js:45:36)
  at Function.Module._load (module.js:325:12)
  at Module.require (module.js:380:17)
  at require (module.js:399:17)
  at Package.module.exports.Package.requireMainModule (/Applications/Atom.app/Contents/Resources/app/src/package.js:651:34)
  at Package.module.exports.Package.activateConfig (/Applications/Atom.app/Contents/Resources/app/src/package.js:251:12)
  at Package.module.exports.Package.activateNow (/Applications/Atom.app/Contents/Resources/app/src/package.js:232:14)
  at /Applications/Atom.app/Contents/Resources/app/src/package.js:221:28
  at Package.module.exports.Package.measure (/Applications/Atom.app/Contents/Resources/app/src/package.js:167:15)
  at Package.module.exports.Package.activate (/Applications/Atom.app/Contents/Resources/app/src/package.js:215:14)
  at PackageManager.module.exports.PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app/src/package-manager.js:464:21)
  at /Applications/Atom.app/Contents/Resources/app/src/package-manager.js:445:29
  at Config.module.exports.Config.transact (/Applications/Atom.app/Contents/Resources/app/src/config.js:350:16)
  at PackageManager.module.exports.PackageManager.activatePackages (/Applications/Atom.app/Contents/Resources/app/src/package-manager.js:439:19)
  at PackageManager.module.exports.PackageManager.activate (/Applications/Atom.app/Contents/Resources/app/src/package-manager.js:422:46)
  at Atom.module.exports.Atom.startEditorWindow (/Applications/Atom.app/Contents/Resources/app/src/atom.js:611:21)
  at Object.<anonymous> (/Applications/Atom.app/Contents/Resources/app/src/window-bootstrap.js:12:8)
  at Object.<anonymous> (/Applications/Atom.app/Contents/Resources/app/src/window-bootstrap.js:23:4)
  at Module._compile (module.js:475:26)
  at Object.loadFile [as .js] (/Applications/Atom.app/Contents/Resources/app/src/6to5.js:140:21)
  at Module.load (/Applications/Atom.app/Contents/Resources/app/node_modules/coffee-script/lib/coffee-script/register.js:45:36)
  at Function.Module._load (module.js:325:12)
  at Module.require (module.js:380:17)
  at require (module.js:399:17)
  at window.onload (file:///Applications/Atom.app/Contents/Resources/app/static/index.js:57:25)

index.js:62 Window load time: 1467ms
```

Does this mean `package.json` should have a dependency on `highlights`? I'm happy to PR that in, but my Node-Fu is weak, so I figured I'd ask first.

**Comment by anthonny** [comment]
Hi @tlberglund !

Could you upgrade Atom ? If noy, you can install an older version using : 
`apm install asciidoc-preview@0.4.2`

Effectively, there is a dependency with highlights.
In the past, i added highlights as a dependency in the package.json, but installation require some development tools like *make* : 

![capture d ecran de 2015-03-10 11 28 05](https://cloud.githubusercontent.com/assets/2006548/6573164/917ef1b0-c718-11e4-8c8c-21a4e9a3c809.png)

That's why i use the highlight module packaged in atom

**Comment by tlberglund** [comment]
@anthonny Oh, that's embarrassing. I thought I was on the latest version, but apparently I had just neglected to restart. I'm in `0.186.0` now, and it works fine. Thanks!

**Comment by anthonny** [comment]
You're welcome!

**Comment by mojavelinux** [comment]
>  I thought I was on the latest version

I got into that same predicament when I was upgrading. In fact, I noticed something interesting to keep in mind when fielding questions. After updating Atom, I uninstalled and reinstalled the AsciiDoc Live Preview plugin. It tried to be smart and install the version I had previously uninstalled. I had to click "Upgrade" on the plugin to get the latest version. When helping people out, make sure that they look for the upgrade button if present to ensure they have the latest version.
